### DISCUSSION TITLE
feature request: Inbox

### DISCUSSION THREAD

**Discussion by Ombucha** [stoatchat/discussions#481] [Feature Suggestions]
### What do you want to see?

The "Inbox" should allow:
- viewing unread messages.
- marking channels, servers and all messages as read.
- viewing recent mentions (add options to exclude `@role` and `@everyone` mentions, and a particular server's mentions).
- collapsing unread messages of a server.
- changing notification settings for the server.
- closing mentions.

### PWA

- [ ] Yes, this feature request is specific to the PWA.

**Comment by zerahfv** [comment]
I like this
